README.md

```diff
@@ -52,18 +52,59 @@ To run the full suite, run: `npm test`.
-source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this
-project's topic/threads here: https://discuss.newrelic.com/c/support-products-agents/node-js-agent/.
+
+Should you need assistance with New Relic products, you are in good hands with several support channels.
+
+**Support Channels**
+
+* [New Relic Documentation](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs): Comprehensive guidance for using our platform
+* [New Relic Community](https://discuss.newrelic.com/tags/nodeagent): The best place to engage in troubleshooting questions
+* [New Relic Developer](https://developer.newrelic.com/): Resources for building a custom observability applications
+* [New Relic University](https://learn.newrelic.com/): A range of online training for New Relic users of every level
+* [New Relic Technical Support](https://support.newrelic.com/) 24/7/365 ticketed support. Read more about our [Technical Support Offerings](https://docs.newrelic.com/docs/licenses/license-information/general-usage-licenses/support-plan).
+
+## Privacy
+
+At New Relic we take your privacy and the security of your information seriously, and are
+committed to protecting your information. We must emphasize the importance of not sharing
+personal data in public forums, and ask all users to scrub logs and diagnostic information
+for sensitive information, whether personal, proprietary, or otherwise.
+
+We define "Personal Data" as any information relating to an identified or identifiable
+individual, including, for example, your name, phone number, post code or zip code,
+Device ID, IP address and email address.
+
+Please review [New Relic’s General Data Privacy Notice](https://newrelic.com/termsandconditions/privacy) for more information.
+
+## Roadmap
+See our [roadmap](https://github.com/newrelic/node-newrelic/blob/main/ROADMAP_Node.md), to learn more about our product vision,
+understand our plans, and provide us valuable feedback. Remove this section if it’s not needed.
 
 ## Contributing
-We encourage your contributions to improve New Relic MySQL instrumentation!  Keep in mind when you
-submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You
-only have to sign the CLA one time per project.
+We encourage your contributions to improve New Relic MySQL instrumentation!  Keep in mind
+when you submit your pull request, you'll need to sign the CLA via the click-through using
+CLA-Assistant. You only have to sign the CLA one time per project.
+If you have any questions, or to execute our corporate CLA, required if your contribution
+is on behalf of a company,  please drop us an email at opensource@newrelic.com.
+
+**A note about vulnerabilities**
+
+As noted in our [security policy](https://github.com/newrelic/.github/blob/master/SECURITY.md), New Relic is committed to
```

Comment:
I'd recommend this point to an easier to maintain location...

https://github.com/newrelic/node-newrelic-mysql/security/policy

That will stay accurate whether or not the other repo's default branch changes or if we were to add a local file for some reason in the future.